### PR TITLE
[FAD-138] Admin Dashboard

### PR DESCRIPTION
**Pull request by sachasmart-weavik**
### Description

https://user-images.githubusercontent.com/98773781/196561274-724e0df1-8944-4078-babe-28dba2cc584f.mov

> Admin Dashboard 
- Admin Dashboard
- Grants artists borrower status
- Borrower can create a pool by creating their borrower contract. 
- Borrower Contract is passed to the `createPool` function. 
### Checklist

- [ ] `CHANGELOG` has been updated (_if appropriate_)
- [ ] Environment variables updated (_if appropriate_)

### CODE REVIEW

File: packages/client3/pages/artist/dashboard/pending-pools-artist.tsx [hunk 1]
```diff
@@ -38,25 +41,44 @@ function PendingPoolArtist() {
...
       console.error("Goldfinch factory couldn't be initialized");
       return;
     }
-    const receipt = await createPool(goldfinchFactory, pool.goalAmount);
-    await updatePoolTransactionHash(pool.id, receipt);
+
+    const borrowerContract = await createBorrowerContract(
+      goldfinchFactory,
+      pool.walletAddress.toLowerCase()
+    );
```
Comment (v1n33th) [inline packages/client3/pages/artist/dashboard/pending-pools-artist.tsx:65]: Just to clarify, Is a new borrower contract created every time we create a pool?

File: packages/client3/utilities/contract.util.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { ContractReceipt } from "ethers";
+import { Result } from "ethers/lib/utils";
+
+export function getLastEventArgs(result: ContractReceipt): Result {
+  const events = result.events;
+  if (events) {
+    const lastEvent = events[events.length - 1];
+    if (lastEvent) {
+      return lastEvent.args as Result;
```
Comment (v1n33th) [inline packages/client3/utilities/contract.util.ts:9]: To clarify isn't the type automatically inferred by Typescript? If it is I would suggest removing explicit typing.

**Review by v1n33th — APPROVED** [review]
Great work....Just few clarifications

File: packages/client3/pages/artist/dashboard/pending-pools-artist.tsx (hunk 1 above)
Comment (sachasmart-weavik) [reply packages/client3/pages/artist/dashboard/pending-pools-artist.tsx:65]: Yes a new contract will be created for each pool that is created.

File: packages/client3/utilities/contract.util.ts (hunk 2 above)
Comment (sachasmart-weavik) [reply packages/client3/utilities/contract.util.ts:9]: I think I was getting a typescript error, but I can double check.

**Review by luansievers — APPROVED** [review]
great work

**Pull request merged**